app/routes.py

```diff
@@ -1,2 +1,286 @@
 from flask import Blueprint
+from .models.customer import Customer
+from .models.video import Video
+from .models.rental import Rental
+from app import db
+from flask import request, make_response, jsonify, Response
+from datetime import datetime, timedelta
+from sqlalchemy import desc, asc
+import os
+import requests
+
+customers_bp = Blueprint("customers", __name__, url_prefix="/customers")
+videos_bp = Blueprint("videos", __name__, url_prefix="/videos")
+rentals_bp = Blueprint("rentals", __name__, url_prefix="/rentals")
+
+def is_int(value):
+    try:
+        return int(value)
+    except ValueError:
+        return False
+
+
+####### CUSTOMER CRUD #########
+
+@customers_bp.route("", methods=["GET"], strict_slashes=False)
+def get_customers():
+    customers = Customer.query.all()
+    
+    customers_response = []
+    for customer in customers:
+        customers_response.append(customer.customer_to_json())
+    return jsonify(customers_response), 200
+
+@customers_bp.route("/<customer_id>", methods=["GET"], strict_slashes=False)
+def get_single_customer(customer_id):
+
+    if not is_int(customer_id):
+        return {
+            "message": f"ID {customer_id} must be an integer",
+            "success": False
+        }, 400
+
+    customer = Customer.query.get(customer_id)
+
+    if customer is not None:
+        return jsonify({"customer": customer.customer_to_json()}), 200
```

Comment:
This code is very close to working, but the return format does not match the format as described in the project docs.
```suggestion
        return jsonify(customer.customer_to_json()), 200
```